src/_common/notification/notification-model.ts

```diff
@@ -103,6 +103,26 @@ export class Notification extends Model {
 		Notification.TYPE_CHARGED_STICKER,
 	];
 
+	static NOTIFICATION_FEED_TYPE_LABELS = {
```

Comment:
$gettext

Reply:
also, add this as a const that gets exported instead of the static thing so that it can tree shake better